.github/workflows/link-pr.yml

```
@@ -0,0 +1,16 @@
+name: Link Checker
+
+on:
+  pull_request
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

В документации всё же так

```suggestion
    - uses: actions/checkout@v2
```